types/selectables/index.d.ts

```diff
@@ -0,0 +1,33 @@
+// Type definitions for selectables 1.4
+// Project: https://github.com/p34eu/Selectables#readme
+// Definitions by: Soner Köksal <https://github.com/renjfk>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = Selectables;
+
+declare class Selectables {
+    constructor(options: Selectables.Options);
```

Comment:
`options` should be optional https://github.com/p34eu/selectables/blob/master/selectables.js#L42

Reply:
I mean `constructor(options?: Selectables.Options);`

Reply:
Ah, I see your point but anyhow it was supposed to be changeable using the object so (;